pldmd/pldmd.cpp

```diff
@@ -241,13 +241,13 @@ int main(int argc, char** argv)
     platformConfigHandler =
         std::make_unique<platform_config::Handler>(PDR_JSONS_DIR);
     std::unique_ptr<oem_fru::Handler> oemFruHandler{};
+    std::unique_ptr<oem_utils::Handler> oemUtilsHandler{};
```

Comment:
please make it as oemUtilsHandler=nullptr

Reply:
Both type of declaration initializes the oemUtilsHandler to nullptr itself, one is explicitly assigns the nullptr, another one uses brace initialization ({}), which is equivalent to default initialization for std::unique_ptr.

Is my understanding correct?

Reply:
yup, just checked both are same, no need to change.